ACE/tests/Process_Manager_Test.cpp

```diff
@@ -437,7 +437,11 @@ run_main (int argc, ACE_TCHAR *argv[])
                               mgr,
                               1,
                               4);
+#if defined (ACE_HAS_CPP11)
```

Comment:
The C++11 template methods do only a convert to ACE _Time_Value and call the regular operations, so even not called explicitly we do test all underlying code, that is why I changed both calls in this test to use the C++11 version.